docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Create a kubernetes cluster on GCP with Gardener
+level: beginner
+category: Getting Started
+scope: app-developer
+---
+
+### Prerequisites
+
+-   You need a GCP account.
+-   You have access to the Gardener dashboard and have permissions to create projects.
+
+1. Go to the Gardener dashboard and create a *Project*.
+
+    <img src="images/new_gardener_project.jpg">
+
+
+2. To check which roles are required by Gardener, choose *Secrets*, and click on the help button <img src="images/help_icon.jpg"> for GCP.
+    <img src="images/gardenergcpsecret1.jpg">
+
+    <img src="images/gardenergcpsecret2.jpg">
+
+3. [Create a new service account in GCP](https://console.cloud.google.com/iam-admin/serviceaccounts) and assign the roles required by Gardener.
+
+    <img src="images/gcpcreateserviceaccount0.jpg">
+
+    <img src="images/gcpcreateserviceaccount1.jpg">
+
+4. To create a key for the service account, choose *Actions* and then *Create key*.
+
+    <img src="images/gcpcreatekey.jpg">
```

Comment:
Same here. Refer the gcp docs. https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys